### PR TITLE
[PATCH v2] Update parser validation test

### PR DESCRIPTION
**Pull request by psavol**
Add more tests for parser validation. Use all existing test packet formats and add new one for SCTP.

### CODE REVIEW

File: test/validation/api/packet/packet.c [hunk 1]
```diff
@@ -41,6 +41,25 @@ static struct udata_struct {
...
+	sizeof(test_packet_arp),
+	sizeof(test_packet_ipv4_icmp),
+	sizeof(test_packet_ipv4_tcp),
+	sizeof(test_packet_ipv4_udp),
+	sizeof(test_packet_vlan_ipv4_udp),
+	sizeof(test_packet_vlan_qinq_ipv4_udp),
+	sizeof(test_packet_ipv6_icmp),
+	sizeof(test_packet_ipv6_tcp),
+	sizeof(test_packet_ipv6_udp),
+	sizeof(test_packet_vlan_ipv6_udp)
```
Comment (lumag) [inline test/validation/api/packet/packet.c]: could you please add IPsec packets from IPsec testsuite?

File: test/validation/api/packet/packet.c [hunk 2]
```diff
@@ -2713,6 +2714,216 @@ static void parse_eth_ipv6_tcp(void)
...
+	CU_ASSERT(odp_packet_parse(pkt[0], offset[0], &parse) == 0);
+	CU_ASSERT(odp_packet_parse_multi(&pkt[1], &offset[1],
+					 num_pkt - 1, &parse) == (num_pkt - 1));
+
+	for (i = 0; i < num_pkt; i++) {
+		CU_ASSERT(odp_packet_has_eth(pkt[i]));
+		CU_ASSERT(odp_packet_has_eth_bcast(pkt[i]));
+		CU_ASSERT(odp_packet_has_arp(pkt[i]));
+		CU_ASSERT(!odp_packet_has_vlan(pkt[i]));
+		CU_ASSERT(!odp_packet_has_ipv4(pkt[i]));
```
Comment (lumag) [inline test/validation/api/packet/packet.c]: And not ipv6 here

**Comment by lumag** [comment]
Reviewed-by: Dmitry Eremin-Solenikov

File: test/validation/api/packet/packet.c (hunk 2 above)
Comment (psavol) [reply test/validation/api/packet/packet.c]: done in v2

File: test/validation/api/packet/packet.c (hunk 1 above)
Comment (psavol) [reply test/validation/api/packet/packet.c]: Done in v2.

**Comment by psavol** [comment]
Ping. Ready to merge. I have another patch set coming on top of this one.

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**